riocli/jsonschema/schemas/package-schema.yaml

```diff
@@ -137,6 +137,58 @@ definitions:
           memory:
             type: number
             minimum: 0
+      livenessProbe:
```

Comment:
The `livenessProbe` should have a `required` section that requires one of `tcp`, `http` or `exec`. Please check the usergroup manifest on how to achieve that.